lib/src/model/settings/board_preferences.dart

```diff
@@ -300,3 +313,17 @@ enum BoardTheme {
           errorBuilder: (context, o, st) => const SizedBox.shrink(),
         );
 }
+
+enum MaterialDifference {
+  materialDifference(label: 'Material difference', visible: true),
+  capturedPieces(label: 'Captured pieces', visible: true),
+  hidden(label: 'Hidden', visible: false);
+
+  const MaterialDifference({
+    required this.label,
+    required this.visible,
```

Comment:
is the `visible` field really needed? A property `bool get hidden => this != MaterialDifference.hidden` would also work (and probably same some memory)